src/Components/Nav/Components/NavSignupModalEmail.js

```diff
@@ -0,0 +1,247 @@
+                </span>
+              </div>
+              <div>
+                {NavSvg.validationNotPassed}
+                <span>최소 8자</span>
+              </div>
+              <div>
+                {NavSvg.validationNotPassed}
+                <span>숫자나 기호를 포함하세요</span>
+              </div>
+            </SignupValidation>
+            <SignupInfoBirth>
+              <h3>생일</h3>
+              <p>
+                만 18세 이상의 성인만 회원으로 가입할 수 있습니다. 생일은 다른
+                에어비앤비 이용자에게 공개되지 않습니다.
+              </p>
+              <BirthSelect width="41.6%">
+                <option>월</option>
+                <option value="1">1월</option>
+                <option value="2">2월</option>
+              </BirthSelect>
+              <BirthSelect width="25%">
+                <option>일</option>
+                <option value="1">1</option>
+                <option value="2">2</option>
+              </BirthSelect>
+              <BirthSelect width="33.3%">
+                <option>년</option>
+                <option value="2020">2020</option>
+                <option value="2019">2019</option>
+              </BirthSelect>
+            </SignupInfoBirth>
+            <div>
+              <p>
+                에어비앤비의 회원 전용 할인, 추천 여행 정보, 프로모션 및 정책
+                변경사항을 이메일로 보내드립니다. 계정 관리의 환경설정 또는
+                프로모션 알림에서 언제든지 메시지 수신을 거부할 수 있습니다.
+              </p>
+              <UserAgreement>
+                <input type="checkbox"></input>
+                <p>에어비앤비에서 보내는 마케팅 메시지를 받고 싶지 않습니다.</p>
+              </UserAgreement>
+            </div>
+            <SignupBtn>가입하기</SignupBtn>
+            <SignupExtra>
+              <span>이미 에어비앤비 계정이 있나요?</span>
+              <SocialTxt font="inherit">로그인</SocialTxt>
+            </SignupExtra>
+          </SignupEmailContent>
+        </SignupEmailWrapper>
+      </SignupEmailContainer>
+    );
+  }
+}
+
+const SignupEmailContainer = styled.div`
+  ${mixin.flexSet}
+  position: fixed;
+  left: 0;
+  top: 0;
+  height: 100%;
+  width: 100%;
+  background: rgba(0, 0, 0, 0.6);
+`;
+
+const SignupEmailWrapper = styled.div`
+  width: 100%;
+  max-width: 568px;
+  border-radius: 12px;
+  box-shadow: rgba(0, 0, 0, 0.28) 0px 8px 28px;
+  background: ${({ theme }) => theme.backgroundColorWhite};
+`;
+
+const SignupEmailHeader = styled.div`
+  ${mixin.flexSet("row", "space-between")}
+  min-height: 64px;
+  padding: 0 24px;
+  border-bottom: ${({ theme }) => theme.borderSet};
+  font-weight: ${({ theme }) => theme.fontBold};
+`;
+
+const SignupEmailContent = styled.div`
+  padding: 24px;
+  overflow: hidden;
+`;
+
+const SocialTxt = styled.button`
+  color: #008489;
+  font-size: ${({ font }) => font};
+`;
+
+const Border = styled.div`
+  margin: 16px 0;
+  text-align: center;
+
+  span {
+    position: relative;
+    padding: 16px;
+    font-size: 14px;
+    font-weight: ${({ theme }) => theme.fontBold};
+    color: ${({ theme }) => theme.fontColorGray};
+
+    &:before {
```

Comment:
```suggestion
    &:before, &:after {
```
이렇게 쓰면 두번 반복하지 않아도 되겠네요